src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import static core.basesyntax.Lottery.getRandomBall;
+
 public class Application {
-    public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+    public static void main(String... args) {
+        for (int i = 0; i < 3; i++) {
+            Ball ball = getRandomBall();
+            System.out.println(ball);
```

Comment:
In situations like this, there is no need to store the object in a local variable if it will be immediately used. Instead, you can pass the expression directly without assigning it to a variable